tests/fuzz/dictionaries/functions.dict

```diff
@@ -1588,6 +1588,7 @@
 "groupBitmapXorResample"
 "groupBitmapXorSimpleState"
 "groupBitmapXorState"
+"groupConcat"
```

Comment:
Good catch, but, unfortunately it is used by [OSSFuzz](https://github.com/google/oss-fuzz/tree/master/projects/clickhouse), not by our fuzzers (because they are AST-based, and can grab all the information from inside the codebase).